iothub/device/samples/DigitalTwinDeviceSamples/PnpHelper/PnpHelper.cs

```diff
@@ -153,6 +154,19 @@ public static (bool, T) GetPropertyFromTwin<T>(TwinCollection collection, string
             return (false, default);
         }
 
+        /// <summary>
+        /// Create a property patch for component-level property updates.
+        /// </summary>
+        /// <param name="propertyKeyValuePairs">The dictionary of property key values pairs that are to be updated.</param>
+        /// <returns>The dictionary containing the property key value pairs for a component-level property update.</returns>
+        public static Dictionary<string, object> CreatePatchForComponentUpdate(Dictionary<string, object> propertyKeyValuePairs)
```

Comment:
I don't understand why this method exists.

Reply:
Discussed offline. Also, this is pending recommendation from sdk board.